config/src/config/mempool_config.rs

```diff
@@ -29,6 +29,8 @@ pub struct MempoolConfig {
     pub broadcast_buckets: Vec<u64>,
     pub eager_expire_threshold_ms: Option<u64>,
     pub eager_expire_time_ms: u64,
+    // length of inbound queue of messages
+    pub max_receiving_messages: usize,
```

Comment:
nit: can we keep this alphabetical (and the same for the ConsensusConfig)? I know some of them aren't, but I think they're wrong/lazy :P